client/app/models/project.js

```diff
@@ -75,10 +75,6 @@ export default class ProjectModel extends Model {
 
   @attr('string') dcpUlurpNonulurp;
 
-  @attr() dcpCommunitydistrict;
```

Comment:
We're removing this because it's not used, right? J/c

Reply:
Correct, I there are three CD fields and CRM and I want to delete all references to the ones we're not using (dcp_communitydistrict and dcp_communitydistricts) to avoid confusion.